.github/workflows/rebuild-all.yaml

```diff
@@ -14,5 +14,5 @@ jobs:
       - name: Repository Dispatch
         uses: peter-evans/repository-dispatch@v2
         with:
-          token: ${{ secrets.PAT_GITHUB_DISPATCH }}
+          token: ${{ secrets.GH_TOKEN_NEXTSTRAIN_BOT_WORKFLOW_DISPATCH }}
```

Comment:
From repository dispatch docs:

> This endpoint requires write access to the repository by providing either:
> - Personal access tokens with `repo` scope. For more information, see "[Creating a personal access token for the command line](https://docs.github.com/articles/creating-a-personal-access-token-for-the-command-line)" in the GitHub Help documentation.

Although the workflow dispatch token has implicit `repo` scope, it might be better to instead use the `GH_TOKEN_NEXTSTRAIN_BOT_REPO` token which has `repo` scope alone.

Reply:
Hmm, now I'm confused why `GH_TOKEN_NEXTSTRAIN_BOT_WORKFLOW_DISPATCH` has additional scope if it's only used to dispatch other GitHub actions workflows.

From [workflows dispatch event docs](https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event):
> You must authenticate using an access token with the `repo` scope to use this endpoint.

Reply:
Oh, good point... I don't have a good reason. We can remove the `workflow` scope and see if it still works.

Reply:
I removed the `workflow` scope and tested with https://github.com/nextstrain/test-github-actions/actions/runs/3660631844. It was able to trigger a [docker-base CI run ](https://github.com/nextstrain/docker-base/actions/runs/3660632757) which I cancelled.

Reply:
Merging this now to fix our automated runs. We can revisit later on whether we should consolidate the two tokens that now have the same `repo` scope.